src/components/Course/index.tsx

```diff
@@ -187,15 +189,21 @@ export default function Course({
                 {
                   icon: expanded ? faAngleUp : faAngleDown,
                   onClick: (): void => prereqControl(false, !expanded),
+                  tooltip: expanded ? 'Hide Sections' : 'View Sections',
```

Comment:
This does not seem to trigger the tooltip. All the other tooltips seem to be working fine 🤔 

![ezgif-1-25645b5b69](https://user-images.githubusercontent.com/54591248/223438771-cc542e00-4586-40a3-9bfd-01b00c52d4cd.gif)